features/bsm/test-util/src/main/java/org/opennms/netmgt/bsm/test/BsmTestUtils.java

```diff
@@ -188,8 +191,16 @@ public static ReductionKeyEdgeResponseDTO toResponseDTO(SingleReductionKeyEdgeEn
     // convert to json
     public static <T> String toJson(T input) {
         Objects.requireNonNull(input);
+
+        final ObjectMapper mapper = new ObjectMapper();
+        final AnnotationIntrospector introspectorPair = AnnotationIntrospector.pair(
+                new JacksonAnnotationIntrospector(),
+                new JaxbAnnotationIntrospector());
+        mapper.setDeserializationConfig(mapper.getDeserializationConfig().withAnnotationIntrospector(introspectorPair));
+        mapper.setSerializationConfig(mapper.getSerializationConfig().withAnnotationIntrospector(introspectorPair));
```

Comment:
Should we add a `createDefaultObjectMapper()` method somewhere? Then we do not have to create the `ObjectMapper` manually all the time.